src/components/calcite-tip-manager/calcite-tip-manager.tsx

```diff
@@ -58,6 +58,8 @@ export class CalciteTipManager {
    */
   @Prop({ reflect: true }) theme: CalciteTheme;
 
+  @Prop({ reflect: true }) tabindex = 0;
```

Comment:
I don't think we want to override default attributes for nodes. The only time I think we're doing that is for the `id` in cases where we must have an `id`.

In this case, I think we need to have a container within the host with a `tabIndex` of `0`.

Reply:
I reviewed the best practices before going forward with this. The best practice suggests that it's alright to use native props on the host, so long as you don't override it if a user sets it. I validated that Stencil's @Props() decorator works this way, if a user puts a tabindex, it will use that instead of the default value here.
https://developers.google.com/web/fundamentals/web-components/best-practices#do-not-override-author-set-global-attributes
[See Connected Callback here](https://developers.google.com/web/fundamentals/web-components/examples/howto-checkbox)

So I think we have the best of both worlds here:
1) We don't create a redundant DOM node when the host will suffice
2) We respect any values set by the end-user, and only resort to our default if none is set.

Reply:
Minor concern (1) is that we are adding a tabIndex to an element automatically and the end user will see this. If it were in the shadowDOM it wouldn't be as exposed to the user. I'm not sure we want them to be able to change this value here.

Minor concern (2) is that this property will be documented in the readme and it really shouldn't be as it isn't a custom attribute.

Reply:
@jcfranco ill let you make the call here

Reply:
Seems like concern #2 is an issue we've run into a few times - I'll submit a feature request with Stencil to be able to opt out a prop from being auto-doc'd

Reply:
I tried moving the tabindex to the tipContainer but it loses focus each time the tip swaps, which leads to a jarring experience.

Reply:
ok